### PR TITLE
Fc dev to Master 

### PR DESCRIPTION
**Pull request by filipceko**
Minor changes to the framevork in order tu support Pyramid Plugin made by Filip Cekovsky 

Extracted Calls and Fanout Measures from the framework - implemented in OverviewPyramid
Updated Sonare version from 6.3 to 6.3.1 
Added Method Return field to the Method Components

### CODE REVIEW

File: framework/src/main/java/framework/java/measurement/MethodReturnVisitor.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    @Override
+    public void visitClass(ClassTree tree){
+        identifier = MeasurementUtils.getClassName(tree, packageName);
+    }
+
+    @Override
+    public void visitMethod(MethodTree tree){
+        if (tree.returnType() == null){
```
Comment (tomaslestyan) [inline framework/src/main/java/framework/java/measurement/MethodReturnVisitor.java:41]: ake pripady toto pokryva? nejak si to nedokazem predstavit

**Review by tomaslestyan — APPROVED** [review]
Dobra praca 👍

File: framework/src/main/java/framework/java/measurement/MethodReturnVisitor.java (hunk 1 above)
Comment (filipceko) [reply framework/src/main/java/framework/java/measurement/MethodReturnVisitor.java:41]: Vytiahne to text returnu metody, potom sa pokusi zistit, ci je returnovana metoda v importoch alebo to nahodou nie je ta v ktorej sa nachadzame, ak ano tak zobere cely ten import (co je vlastne fully qualified name), ak nie, vezme co nasiel v dekleracii metody a ulozi iba to. Tie nie fully qualified potom dalej spracovavam vo svojom plugine ...

Comment (filipceko) [reply framework/src/main/java/framework/java/measurement/MethodReturnVisitor.java:41]: Jaj uz rozumiem o co slo, to su konstruktory, ktore nemaju ako vratnu hodnotu ani hodnotu void ale vysledkom je vlastne instancia tej triedy.

**Pull request merged**